source/faq.txt

```diff
@@ -261,46 +261,53 @@ socket connections.
 If you experience unexpected network behavior, here
 are some things to check:
 
-#. The firewall should send a ``FIN packet`` when closing a socket,allowing the driver to detect that the socket is closed.
+#. The firewall should send a ``FIN packet`` when closing a socket,
+   allowing the driver to detect that the socket is closed.
 #. The firewall should allow ``keepAlive`` probes.
 
 How Can I Prevent a Slow Operation From Delaying Other Operations?
 ------------------------------------------------------------------
 
-You can prevent an operation from slowing down your application by
-specifying the size of the connection pool, which is the cache of
-connections that the driver maintains at any time. By increasing the
-maximum connection pool size, you can reduce application latency.
-
-To control the maximum size of a connection pool, you can set the
-``maxPoolSize`` option in the :ref:`connection options
-<node-connection-options>`. The default value of ``maxPoolSize`` is
-``100``. If the number of in-use connections to a server reaches
-``maxPoolSize``, the next request to that server will wait 
-until a connection becomes available. The following code sets
+Before you make changes to your connection configuration or application,
+you should determine which operations could be causing delays. You can
+investigate the performance of in-progress operations by using the
+following strategies:
+
+- Enable the database profiler on your deployment. To learn more, see
+  :manual:`Database Profiler </tutorial/manage-the-database-profiler/>`
+  in the Server manual.
+- Run the ``db.currentOp()`` :manual:`shell command </reference/method/>`. To learn more, see the 
+  :manual:`db.currentOp() </reference/method/db.currentOp/>`
+  documentation in the Server manual.
+- Enable connection pool monitoring. To learn more, see
+  :ref:`node-connection-pool-monitoring`.
+
+Once you determine how and why your operations are delayed, if possible,
```

Comment:
Suggestion:
Omit "how and why" to fit better in the suggested flow of this section.